conf.py

```diff
@@ -184,7 +184,7 @@
 # (source start file, target name, title, author, documentclass [howto/manual]).
 latex_documents = [
   ('index', 'gcc.tex', u'gcc Documentation',
-   u'gcc peeps', 'manual'),
```

Comment:
Authors should presumably be something like "GCC Contributors"

Reply:
Yep, fixed now.